lib/authz/permissions.go

```diff
@@ -365,6 +365,15 @@ func CheckIPPinning(ctx context.Context, identity tlsca.Identity, pinSourceIP bo
 		}
 		return ErrIPPinningMismatch
 	}
+	if clientPort == "0" {
```

Comment:
Can you please link the places where we replace the port and a short description why we do it